### YamlMime:PythonClass
uid: azure.mgmt.costmanagement.models.ErrorResponse
name: ErrorResponse
fullName: azure.mgmt.costmanagement.models.ErrorResponse
module: azure.mgmt.costmanagement.models
inheritances:
- azure.mgmt.costmanagement._serialization.Model
summary: "Error response indicates that the service is not able to process the incoming\
  \ request. The\nreason is provided in the error message.\n\nSome Error responses:\n\
  \n* 429 TooManyRequests - Request is throttled. Retry after waiting for the time\
  \ specified in the \n\n\"x-ms-ratelimit-microsoft.consumption-retry-after\" header.\n\
  \n* 503 ServiceUnavailable - Service is temporarily unavailable. Retry after waiting\
  \ for the time \n\nspecified in the \"Retry-After\" header."
constructor:
  syntax: 'ErrorResponse(*, error: _models.ErrorDetails | None = None, **kwargs: Any)'
  keywordOnlyParameters:
  - name: error
    description: The details of the error.
    types:
    - <xref:azure.mgmt.costmanagement.models.ErrorDetails>
variables:
- description: The details of the error.
  name: error
  types:
  - <xref:azure.mgmt.costmanagement.models.ErrorDetails>
